vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddBiblioBlockController.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.asu.diging.vspace.core.model.IBiblioBlock;
+import edu.asu.diging.vspace.core.model.impl.BiblioBlock;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+
+@Controller
+public class AddBiblioBlockController {
+
+    @Autowired
+    private IContentBlockManager contentBlockManager;
+
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/bibliography", method = RequestMethod.POST)
```

Comment:
since you use `reference/add` below, I would use `references` here

Reply:
i do not see "reference/add" so i think this is already addressed

Reply:
`AddReferenceController` uses `references/add`, so here it should be `references` not `bilbiography`

Reply:
this is AddBiblioBlockController, so shouldn't it be just bibliography? it add bibliography block to the slides

Reply:
ah yes! that makes sense.